### PR TITLE
 [session01] 김재성 앙상블 과제, 딥러닝1 사전과제 제출합니다.

### PR DESCRIPTION
**Pull request by i-am-jsung**
### 🔍 어려웠던 점 및 새로 알게 된 점 간단히 정리
파이프라인으로 처리할 수 있다는 걸 새롭게 알게 되었습니다. 회귀와 분류 문제에 모두 앙상블로 다뤄볼 수 있어서 너무 좋았습니다. 성능 높이는게 어려웠지만 다양한 하이퍼파라미터를 변경해가면서 의미를 파악할 수 있었습니다. GridSearch"CV" 넘 오래 걸리고 잘 안되고 어렵네요  

### 🔍 질문 및 하고 싶은 말
scaler.fit_transform하고 train_test_split하는게 데이터 누수라는 주장이 있습니다! 성능에서는 별로 차이가 안 나긴 하던데...
최종 성능 기준은 Accuracy보다는 F1-score / ROC-AUC / PR-AUC로 확인하는 것이 안전하다고 하는데 어렵네요.
준비하느라 정말 수고하셨습니다 ~ ! ~ !

---
### ✅ PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.
- [x] 본인 세션별 브랜치 → develop 브랜치 로 PR 합니다.
[](url)

### CODE REVIEW

**Comment by neulbokim** [comment]
오호.. 재성 오빠 덕에 새로운 것을 알게 되었네요 😮😮 accuracy 말고 F1-score / ROC-AUC / PR-AUC로 다시 순위 매겨야 하려나요?!!

이번 과제에서 soft voting으로 열심히 돌려주셔서 아주 좋은 결과가 나온 듯해요!! ㅎㅎ 수고하셨습니다 👍👍 그리고 랜덤 서치 말고 손으로 하이퍼파라미터 찾은 거예요..?? 😲😲 정말 대단하다.......

이번 과제 너무 수고 많으셨습니다~!! 2차시 세션도 파이팅 🔥🔥
